api/RTCDataChannel.json

```diff
@@ -1304,6 +1304,58 @@
             "deprecated": true
           }
         }
+      },
+      "transferable": {
+        "__compat": {
```

Comment:
@sideshowbarker Worth adding a spec url to the glossary definition https://developer.mozilla.org/en-US/docs/Glossary/Transferable_objects ?

Were you planning on adding this for the other cases or did you want me to/share the load?

Reply:
> @sideshowbarker Worth adding a spec url to the glossary definition [developer.mozilla.org/en-US/docs/Glossary/Transferable_objects](https://developer.mozilla.org/en-US/docs/Glossary/Transferable_objects) ?

👍 Yes

> Were you planning on adding this feature for the other cases or did you want me to/share the load?

If you could that’d be great

Reply:
I realised adding the feature for other transferrable things would be a tough job. We'd need to write some test code that tried it out. Or find someone who knows. Upshot, not signing up for that :-0